### ISSUE TITLE
Part 6 Lesson 1.25 Encoding Issue

### ISSUE THREAD

**Issue by carsten-grimm** [udacity/isdc_issues_tracker#123]
When testing my solution to the Quiz, all tests pass. When I submit the solution, the following error is displayed: 
```
Traceback (most recent call last):
  File "vm_main.py", line 33, in 
    import main
  File "/tmp/vmuser_pbthsxymtd/main.py", line 2, in 
    import aiMain
  File "/tmp/vmuser_pbthsxymtd/aiMain.py", line 7
SyntaxError: Non-ASCII character '\xc3' in file /tmp/vmuser_pbthsxymtd/aiMain.py on line 8, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details```

I am not using any unicode characters in my solution as far as I know.

**Comment by andrewpaster** [comment]
Hey apologies for the delay in getting back about this. Is Part 6 referring to Navigating Data Structures? Part 6 1.25 for me is Navigating Data Structures -> How to Solve Problems -> 25. Making Progress is Good. It looks like that node doesn't have a quiz attached to it, so I wasn't sure which quiz was being referred to.

**Comment by carsten-grimm** [comment]
Sorry about this, I should have included the title. There are two new parts that were included after I wrote the reference to 6.1.25. I was referring to "Quiz: Define daysBetweenDates" that is now 6.1.27. 

I checked the Quiz again and it appears to be working now.

**Comment by andrewpaster** [comment]
cool thanks for checking and for getting back to me. I'll close this.
